index.js

```diff
@@ -7,9 +7,13 @@
 const crypto = require('crypto')
 const qs = require('qs')
 const Promise = require('bluebird')
-const sqs = require('sqs')
+const AWS = require('aws-sdk')
+const P = require('bluebird')
 
-const { AUTH, SQS_SUFFIX, PROVIDER } = process.env
+AWS.config.setPromisesDependency(P)
```

Comment:
I don't actually need to do this step to use `bluebird` as the Promise impl, but I did.